packages/@lwc/template-compiler/src/__tests__/fixtures/parsing-errors/duplicate-default-slot-light/actual.html

```diff
@@ -0,0 +1,4 @@
+<template lwc:render-mode="light">
+    <slot></slot>
+    <slot></slot>
+</template>
```

Comment:
I also added some tests for light DOM mode to ensure that light DOM has the same behavior as shadow DOM.